packages/webview_flutter/tizen/src/webview_factory.cc

```diff
@@ -6,51 +6,42 @@
 
 #include <app_common.h>
 #include <flutter/encodable_value.h>
+#include <flutter/message_codec.h>
 
 #include <string>
 #include <variant>
 
 #include "log.h"
 #include "lwe/LWEWebView.h"
+#include "webview.h"
 
-WebViewFactory::WebViewFactory(flutter::PluginRegistrar* registrar,
-                               flutter::TextureRegistrar* texture_registrar)
-    : PlatformViewFactory(registrar), texture_registrar_(texture_registrar) {
+static std::string GetAppDataPath() {
   char* path = app_get_data_path();
-  std::string path_string;
-  if (!path || strlen(path) == 0) {
-    path_string = "/tmp/";
-  } else {
-    path_string = path;
-    free(path);
-    path = nullptr;
+  if (!path) {
+    return "/tmp/";
   }
-  LOG_DEBUG("application data path : %s\n", path_string.c_str());
-  std::string local_storage_path =
-      path_string + std::string("StarFish_localStorage.db");
-  std::string cookie_path = path_string + std::string("StarFish_cookies.db");
-  std::string cache_path = path_string + std::string("Starfish_cache.db");
+  std::string result = std::string(path);
+  free(path);
+  return result;
+}
+
+WebViewFactory::WebViewFactory(flutter::PluginRegistrar* registrar)
+    : PlatformViewFactory(registrar) {
+  texture_registrar_ = registrar->texture_registrar();
+
+  std::string data_path = GetAppDataPath();
+  std::string storage_path = data_path + "StarFish_localStorage.db";
```

Comment:
I think local_storage_path is more appropriate. because it represent path of [local storage](https://developer.mozilla.org/ko/docs/Web/API/Window/localStorage)

Reply:
Okay :)